### PR TITLE
Add Extra Logging

### PR DESCRIPTION
**Pull request by Maithem**
Added extra logging to the MultiCheckpointWriter.

### CODE REVIEW

**Comment by corfudb-performance** [comment]
Results automatically generated by _CorfuDB Benchmark Framework_ to assess the performance of this pull request for commit **9836c78**.

*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 10 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Iterator workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Iterator workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Iterator workload, 10 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 10 threads, Disk mode
 
 An interactive dashboard with Pull Request Performance Metrics for ALL cluster types and numbers of threads in run, is available at:
 [Pull Request #891 Graphs ](https://snapshot.raintank.io/dashboard/snapshot/pQ22xVfpk7eCrdt6TmkJL4KMvRVDy40S)

**Comment by corfudb-performance** [comment]
Results automatically generated by _CorfuDB Benchmark Framework_ to assess the performance of this pull request for commit **dda571e**.

*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 10 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Iterator workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Iterator workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Iterator workload, 10 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 10 threads, Disk mode
 
 An interactive dashboard with Pull Request Performance Metrics for ALL cluster types and numbers of threads in run, is available at:
 [Pull Request #891 Graphs ](https://snapshot.raintank.io/dashboard/snapshot/lsG9O3MX20LWCOb4tRBborfGzK8z2TTr)

File: runtime/src/main/java/org/corfudb/runtime/MultiCheckpointWriter.java [hunk 1]
```diff
@@ -75,10 +75,12 @@ public long appendCheckpoints(CorfuRuntime rt, String author,
         log.trace("appendCheckpoints: author '{}' at globalAddress {} begins",
                 author, globalAddress);
 
+        log.info("appendCheckpoints: appending checkpoints for {} maps", maps.size());
+        long cpStart = System.currentTimeMillis();
         try {
             for (ICorfuSMR<Map> map : maps) {
                 UUID streamId = map.getCorfuStreamID();
-
+                long mapCpStart = System.currentTimeMillis();
```
Comment (no2chem) [inline runtime/src/main/java/org/corfudb/runtime/MultiCheckpointWriter.java]: mark final

File: runtime/src/main/java/org/corfudb/runtime/MultiCheckpointWriter.java [hunk 2]
```diff
@@ -75,10 +75,12 @@ public long appendCheckpoints(CorfuRuntime rt, String author,
         log.trace("appendCheckpoints: author '{}' at globalAddress {} begins",
                 author, globalAddress);
 
+        log.info("appendCheckpoints: appending checkpoints for {} maps", maps.size());
+        long cpStart = System.currentTimeMillis();
```
Comment (no2chem) [inline runtime/src/main/java/org/corfudb/runtime/MultiCheckpointWriter.java]: mark final

**Comment by Maithem** [comment]
Addressed changes.

**Pull request merged**